app/utils/factory.py

```diff
@@ -1,7 +1,30 @@
+from datetime import datetime
+from typing import ClassVar
+
+import pytz
 from pydantic import BaseModel, Field
 from uuid import uuid4
 
+def get_current_datetime_with_tz():
+    tz = pytz.timezone('Asia/Ho_Chi_Minh')
+    return datetime.now(tz=tz)
 
 # Util classes for schemas
 class UUID4Factory(BaseModel):
     id: str = Field(default_factory=lambda: str(uuid4()))
+
+class CurrentTimeFactory(BaseModel):
```

Comment:
This class includes two files `created_at`, `updated_at`, so I'm wondering if the class name is descriptive.